inc/order.class.php

```diff
@@ -2048,8 +2048,8 @@ public static function showForBudget($budgets_id)
                 );
 
                 //if state is cancel do not decremente total already use
-                if ($data['plugin_order_orderstates_id'] < 5) {
-                     $total += $prices["priceHT"];
+                if (!in_array($data['plugin_order_orderstates_id'], [PluginOrderOrderState::CANCELED, PluginOrderOrderState::DRAFT, PluginOrderOrderState::WAITING_FOR_APPROVAL])) {
+                     $total += $prices["priceTTC"];
```

Comment:
```suggestion
                if ($data['plugin_order_orderstates_id'] != PluginOrderOrderState::CANCELED) {
                     $total += $prices["priceTTC"];
```

Reply:
Based on the code and its usage within the community, it is simply necessary not to process orders with a `Cancel` status.

Reply:
Actually `PAID (7)` and `DELIVRY (5)` statut are not processes